library/src/main/java/com/schibsted/spain/barista/cleardata/FileOperations.java

```diff
@@ -0,0 +1,37 @@
+package com.schibsted.spain.barista.cleardata;
+
+import android.content.Context;
+import java.io.File;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+public class FileOperations {
+
+  static void deleteAllFiles(Context appContext) {
+    for (File file : getFiles(appContext)) {
+      file.delete();
+    }
+  }
+
+  private static Collection<File> getFiles(Context appContext) {
+    File filesDir = appContext.getFilesDir();
```

Comment:
Yes, let's keep that really important discussion in just one place, to keep brain sane :·D